### PR TITLE
[Proposal] Tech Debt Week Process Design Doc

### PR DESCRIPTION
**Pull request by electronjoe**
Signed-off-by: Scott Harrison Moeller <smoeller@fb.com>

### CODE REVIEW

**Comment by electronjoe** [comment]
This is the design document for #6265

File: docs/readmes/proposals/p015_tech_debt_week.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      2. Post it in Slack [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN)
+      3. You can convert to draft if you mistakenly create full fledge PR (`Still in progress? Convert to draft`)
+   2. Review Timeliness and Tone (for both Draft and Final)
+      1. Expect [Timely review](#issues-lead-response-time)
+      2. With a [Spheherd menality](https://github.com/magma/magma/blob/master/docs/readmes/contributing/contribute_codeowners.md#shepherds-not-gatekeepers)
+      3. Feel free to ask for precise assistance or examples (e.g. a GH gist with example code on an ask)
+   3. The [Issue Lead](#magma-issue-lead) will aim to make contact daily if they haven't heard from you
+   4. Apply any directional changes advised to draft PR
+6. When a final PR is ready for review
+   1. click `Ready for Review` to convert from Draft PR to PR
```
Comment (themarwhal) [inline docs/readmes/proposals/p015_tech_debt_week.md]:
```suggestion
   1. Click `Ready for Review` to convert from Draft PR to PR
```

File: docs/readmes/proposals/p015_tech_debt_week.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+### TDW On-call Contact
+
+- [Video conference](TBD LINK) monitored by [TDW On-Call](#magma-tdw-on-call)
+- Slack channel [#project-tech-debt-week-apr-2021](https://magmacore.slack.com/archives/C0202TW1VRN) monitored by [TDW On-Call](#magma-tdw-on-call)
+- Page: email the TDW On-call at tdw@magma.pagerduty.com
+  - Provide a preferred contact (e.g. slack id, github id, ...)
+
+### TDW On-Call Response time
+
+During TDW, We ask that the following interactions and actions are given `highest work priority` - ideally with a response time of `< 1 hour`.  This is to ensure maximum velocity for our external contributors during this tight week work window - and to ensure we are efficient with their donated time.
```
Comment (themarwhal) [inline docs/readmes/proposals/p015_tech_debt_week.md]: Should we specify time too? like response time <1 hour between 9AM pacific and 3PM pacific?

File: docs/readmes/proposals/p015_tech_debt_week.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      2. Move GH Issue back to available stack
+         1. Assigned -> Unassigned
+
+## Magma Issue Lead
+
+A Magma Issue Lead is the designated individual who has filed a GH Issue deemed appropriate for TDW inclusion.  These individuals may be a CODEOWNER for the relevant code base, but at least have a clear understanding of the issue's resolution and has strong communications channels with the CODEOWNERS implicated.  Determining who is Issue Lead will occur between TDW On-Call and relevant Mamga core team members.
+
+### Issues Lead Response time
+
+During TDW, We ask that the following interactions and actions are given `highest work priority` - ideally with a response time of `< 1 hour`.  This is to ensure maximum velocity for our external contributors during this tight week work window - and to ensure we are efficient with r donated time.
```
Comment (themarwhal) [inline docs/readmes/proposals/p015_tech_debt_week.md]: with <??> donated time? The last clause is probably not what you intended?

File: docs/readmes/proposals/p015_tech_debt_week.md (hunk 2 above)
Comment (electronjoe) [reply docs/readmes/proposals/p015_tech_debt_week.md]: Good point! Done.

File: docs/readmes/proposals/p015_tech_debt_week.md (hunk 3 above)
Comment (electronjoe) [reply docs/readmes/proposals/p015_tech_debt_week.md]: Trimmed!

**Review by themarwhal — APPROVED** [review]
lgtm!

**Pull request merged**